AmplifyPlugins/Core/AWSPluginsCore/Model/Support/ModelSchema+Extension.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+
+extension ModelSchema {
```

Comment:
this no longer needs to be a generic `+Extension` for both DataStore and API since for DataStore, we are using the columnName from `ModelField: SQLColumn` that returns a column name specific for SQL statements, ie. quoting the fields and prepending namespace. So this can be moved over to a more GraphQL specific extension